### ISSUE TITLE
Lightforge no longer to be updated

### ISSUE THREAD

**Issue by jvook** [supertriodo/Arena-Tracker#125]
FYI

![Screenshot_20210126-105303](https://user-images.githubusercontent.com/17225171/105869187-f72bbe80-5fc4-11eb-9d81-c03b43a8b72a.png)

**Comment by supertriodo** [comment]
Yes, I've already spoken with adwcta. I'll remove their tier list from AT.
Thanks.

**Comment by supertriodo** [comment]
https://github.com/supertriodo/Arena-Tracker/releases/tag/v9.1
